infra/charts/feast-feature-server/README.md

```diff
@@ -24,26 +24,27 @@ See [here](https://github.com/feast-dev/feast/tree/master/examples/python-helm-d
 
 ## Values
 
-| Key | Type | Default | Description |
-|-----|------|---------|-------------|
-| affinity | object | `{}` |  |
-| feature_store_yaml_base64 | string | `""` | [required] a base64 encoded version of feature_store.yaml |
-| fullnameOverride | string | `""` |  |
-| image.pullPolicy | string | `"IfNotPresent"` |  |
-| image.repository | string | `"feastdev/feature-server"` | Docker image for Feature Server repository |
-| image.tag | string | `"0.37.1"` | The Docker image tag (can be overwritten if custom feature server deps are needed for on demand transforms) |
-| imagePullSecrets | list | `[]` |  |
-| livenessProbe.initialDelaySeconds | int | `30` |  |
-| livenessProbe.periodSeconds | int | `30` |  |
-| nameOverride | string | `""` |  |
-| nodeSelector | object | `{}` |  |
-| podAnnotations | object | `{}` |  |
-| podSecurityContext | object | `{}` |  |
-| readinessProbe.initialDelaySeconds | int | `20` |  |
-| readinessProbe.periodSeconds | int | `10` |  |
-| replicaCount | int | `1` |  |
-| resources | object | `{}` |  |
-| securityContext | object | `{}` |  |
-| service.port | int | `80` |  |
-| service.type | string | `"ClusterIP"` |  |
-| tolerations | list | `[]` |  |
\ No newline at end of file
+| Key | Type | Default                     | Description                                                                                                 |
+|-----|------|-----------------------------|-------------------------------------------------------------------------------------------------------------|
```

Comment:
why is the formatting of these 2 lines changed? if not functionally necessary, please revert.

Reply:
it seems like the formatting changed throughout this file... please revert

Reply:
I have added `feast_mode` filed in that table with description which cause reformatting

Reply:
can you revert it though and add that one line without the re-formatting? this is important because in case we ever decide to submit this upstream.

Reply:
reverted